scipy/linalg/tests/test_lapack.py

```diff
@@ -703,6 +703,9 @@ def test_hetrd(self):
             # query lwork
             lwork, info = hetrd_lwork(n)
```

Comment:
Can we avoid calculating ``lwork`` twice? I haven't looked at it in detail, but I suspect, info is only a status code or similar.

Reply:
Yes it is just testing whether `hetrd_lwork` working correctly or not.